fs_storage/models/fs_storage.py

```diff
@@ -130,6 +130,17 @@ def __init__(self, env, ids=(), prefetch_ids=()):
         help="Relative path to the directory to store the file"
     )
 
+    store_connection = fields.Boolean(
```

Comment:
I wonder if this is needed and if yes, under which circumstances.
Also, is worth to mention in the help that the cache is only by thread.

Reply:
@simahawk you are right, I have not checked it but probably with the invalidation of the cache of the FS it will be enough. I will remove it and test.